jwst_reffiles/bad_pixel_mask/bad_pixel_mask.py

```diff
@@ -168,19 +179,34 @@ def find_bad_pix(input_files, dead_search=True, low_qe_and_open_search=True, dea
     # Create mean and stdev images
     mean_img, stdev_img = mean_stdev_images(input_exposures, sigma=sigma_threshold)
 
+    # Exclude reference pixels
+    mean_img = science_pixels(mean_img)
+    stdev_img = science_pixels(stdev_img)
+
     # Create smoothed version of mean image
-    smoothed_image = smooth(mean_img, box_width=smoothing_box_width)
+    if normalization_method.lower() == 'smoothed':
+        smoothed_image = smooth(mean_img, box_width=smoothing_box_width)
+    elif normalization_method.lower() == 'mean':
+        img_mean, img_dev = image_stats(image, sigma=3.)
```

Comment:
should this be image_stats(mean_img, sigma=3.)? The code crashes if I try to use normalization_method='mean' because 'image' is not defined.

Reply:
Good catch. I'll update this too.